CHANGES.md

```diff
@@ -1,3 +1,45 @@
+## edge-20.2.2
+
+This release includes the results from continued profiling & performance
+analysis on the Linkerd proxy. In addition to modifying internals to prevent
+unwarranted memory growth, new metrics were introduced to aid in debugging and
+diagnostics. Also, Linkerd's CNI plugin is out of experimental!
+
+* CLI
+  * Added support for label selectors in the `linkerd stat` command (thanks
+    @mayankshah1607!)
+  * Added scrolling functionality to the `linkerd top` output (thanks
+    @kohsheen1234!)
+  * Fixed bug in `linkerd metrics` that was causing a panic when port-forwarding
+    failed (thanks @mayankshah1607!)
+  * Added check to `linkerd check` verifying the number of replicas for Linkerd
+    components in HA (thanks @mayankshah1607!)
+  * Unified trust anchors terminology across the CLI commands
+  * Removed some messages from `linkerd upgrade`'s output that are no longer
+    relevant (thanks @supra08!)
+
+* Controller
+  * Added support for configuring service profile retries
+    `(x-linkerd-retryable)` via OpenAPI spec (thanks @kohsheen1234!)
+  * Improved traffic split metrics so sources in all namespaces are shown, not
+    just traffic from the traffic split's own namespace
+  * Improved linkerd-identity's logs and events to help diagnosing certificate
+    validation issues (thanks @mayankshah1607!)
+
+* Proxy
+  * Added `request_errors_total` metric exposing the number of requests that
+    receive synthesized responses due to proxy errors
+
+* Helm
+  * Added a new `enforcedHostRegexp` value to allow configuring the linkerd-web
```

Comment:
s/value/variable - the way I look at it: we added the new variable to make the value configurable.